packages/core/src/Renderer.ts

```diff
@@ -98,8 +98,6 @@ export class Renderer extends AbstractRenderer
      *   resolutions other than 1.
      * @param {boolean} [options.antialias=false] - Sets antialias. If not available natively then FXAA
```

Comment:
Missed a spot here: _"If not available natively then FXAA antialiasing is used"_.

Reply:
Yup! Good catch